src/Avatar/Avatar.js

```diff
@@ -8,9 +8,14 @@ import StyledAvatarText from './StyledAvatarText';
 
 const AVATAR_COLORS = [
   'palette.blue',
+  'palette.cyan',
+  'palette.green',
+  'palette.orange',
+  'palette.pink',
+  'palette.purple',
   'palette.red',
-  'palette.yellow',
-  'palette.green'
+  'palette.teal',
+  'palette.yellow'
```

Comment:
Would you be opposed to importing theme/colors/palette and grabbing the keys to construct the list?

Reply:
Good call!